config/rbac/role.yaml

```diff
@@ -1,13 +1,14 @@
-
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  creationTimestamp: null
   name: manager-role
 rules:
 - nonResourceURLs:
   - /metrics
+  verbs:
+  - get
+- nonResourceURLs:
   - /metrics/slis
```

Comment:
the diff is misleading, the actual change is the addition of lines 11-14 due to the additional kubebuilder tag